.github/workflows/release.yml

```
@@ -47,12 +48,15 @@ jobs:
         roleId: ${{ secrets.VAULT_ROLE_ID }}
         secretId: ${{ secrets.VAULT_SECRET_ID }}
 
-    - name: Publish Docker Image
+    - name: Build Profiler Docker Image
+      continue-on-error: true #continue for now until we see it working in action
+      run: |
+        docker build . -t docker.elastic.co/observability/apm-agent-dotnet:${{ steps.bootstrap.outputs.agent-version }} \
+            --build-arg AGENT_ZIP_FILE=build/output/elastic_apm_profiler_${{ steps.bootstrap.outputs.agent-version }}-linux-x64.zip
+    - name: Push Profiler Docker Image
+      continue-on-error: true #continue for now until we see it working in action
       run: |
-        chmod +x .ci/linux/push_docker.sh
-        .ci/linux/push_docker.sh
-      env:
-        TAG_NAME: ${{ github.ref_name }}
+        docker push docker.elastic.co/observability/apm-agent-dotnet:${{ steps.bootstrap.outputs.agent-version }}
```

what about using something like this
```suggestion
        for i in $(seq 1 3); do [ $i -gt 1 ] && sleep 15; docker push docker.elastic.co/observability/apm-agent-dotnet:${{ steps.bootstrap.outputs.agent-version }} && s=0 && break || s=$?; done; (exit $s)
```

I took the loop/sleep/retry from [here](https://unix.stackexchange.com/a/82610)

---

Love it!